demo.sh

```diff
@@ -1,23 +1,33 @@
+#/bin/bash
+
+URL=${URL:-localhost:8080}
+
+RESP=$(curl -D- "$URL/auth/v1.0" -H "X-Auth-User: test:tester" -H "X-Auth-Key: testing" 2>1)
+TOKEN=$(echo "$RESP" | awk '/X-Auth-Token:/ {printf "%s", $2}' | tr -d '\r')
+STORAGE_URL=$(echo "$RESP" | awk '/X-Storage-Url:/ {printf "%s", $2}' | tr -d '\r')
```

Comment:
fancy-ist `awk | tr` I'd seen in awhile - KUDOS

Reply:
the documentation makes it sound like the `printf "%s, %2` should strip the `\r` on it's own, but it *doesnt* 😭